lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -864,7 +864,7 @@ function dosomething_reportback_get_reportback_total_by_nid($nid, $type = 'quant
     $query->fields('rb', array('quantity'));
     $query->addExpression('SUM(quantity)', 'quantity');
     $result = $query->execute()->fetchAll();
-    $quantity = $result[0]->quantity;
+    $quantity = (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quantity_override') + (int) $result[0]->quantity;
```

Comment:
I know there are a couple of places in the code where these two vars are already added together. This will cause the result to be double if we don't remove it from there. 
Also I'm a little nervous about having this query return the result of the summary and not having the two counts just added together in the campaign.theme.inc file...